common/src/java/org/apache/hive/http/HttpServer.java

```diff
@@ -136,6 +136,11 @@ public class HttpServer {
 
 
   private final String name;
+  /**
+   * The list of web application contexts associated with the server.
+   * The first web application context at index 0 is the default context added during web server creation.
```

Comment:
wouldn't it be better to define an instance var for the `default` / `root` WebAppContext`?